### ISSUE TITLE
Extension modules - update MSVC for standalone module

### ISSUE THREAD

**Issue by zooba** [python/devguide#1394]
The page https://devguide.python.org/developer-workflow/extension-modules/#updating-msvc-project-files has recently been updated with fantastic content.

However, it currently only shows how to add a _built-in_ module for Windows, and not a separate extension module.

We should add a second section that adds a new `.vcxproj` for the new extension module (or more likely copies a recommended existing one), along with the other changes necessary to ensure the new module is correctly built and distributed.

**Comment by picnixz** [comment]
Oh thank you for this Steve. I indeed see that I only mentioned the `pythoncore.vcxproj` file, so I'll try to come up with something for a separate extension module. I never used MSVC and the section I wrote about MSVC was mostly a guess hence the confusion on my side.

**Comment by zooba** [comment]
Yeah, no worries. This is as much a note to myself (or an invitation to anyone else who wants to contribute) as anything else. Certainly not meant as a criticism of your work so far!
